lib/api/database/user_data.dart

```diff
@@ -28,7 +28,7 @@ class UserData {
   UserData(this.client) {
     account = Account(client);
     storage = Storage(client);
-    database = Database(client);
+    database = Databases(client, databaseId: 'YOUR DATABASE ID');
```

Comment:
Since `databaseId` is gonna be constant. Let's put this in `ApiInfo` class

so here it should be: 
`database = Databases(client, databaseId: ApiInfo.databaseId);`

In the readme under [Setup Project Dashboard](https://github.com/2002Bishwajeet/no_signal/blob/master/README.md#setup-project-dashboard), add the following line,

`static const String databaseId = "[YOUR_DATABASE_ID]";`

Reply:
Will do 💯